### PR TITLE
Add CLIP vectorizer module

### PR DESCRIPTION
**Pull request by kunalvirwal**

### CODE REVIEW

File: modules/CLIP-vectorizer/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+myenv2/
```
Comment (h4shk4t) [inline modules/CLIP-vectorizer/.dockerignore]: Why is this here? Get rid of it

Comment (kunalvirwal) [reply modules/CLIP-vectorizer/.dockerignore]: Done
